### PR TITLE
{math}[intel/2020a] Harminv v1.4.1

### PR DESCRIPTION
**Pull request by ekieffer**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ekieffer** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-002 - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/e88abb2592f4b161f0a329c51a7ea8d3 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node378.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/83000ab11fef4296235643cc67be037a for a full test report.

File: easybuild/easyconfigs/h/Harminv/Harminv-1.4.1-intel-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+source_urls = ['https://github.com/stevengj/harminv/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['e1b923c508a565f230aac04e3feea23b888b47d8e19b08816a97ee4444233670']
+
+configopts = "--with-pic  --with-blas=mkl_em64t --with-lapack=mkl_em64t --enable-shared"
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['bin', 'lib']
```
Comment (lexming) [inline easybuild/easyconfigs/h/Harminv/Harminv-1.4.1-intel-2020a.eb:23]: The sanity checks can be improved
```suggestion
    'files': ['bin/harminv', 'lib/libharminv.a', 'lib/libharminv.%s' % SHLIB_EXT],
    'dirs': ['include']
```

**Review by lexming — CHANGES_REQUESTED** [review]
This PR works well. I would just make a few changes to the sanity checks. Please check below

**Comment by smoors** [comment]
@boegelbot please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node378.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/f884f3d0a59edef9a568f33ea3a0d490 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @ekieffer!

**Pull request merged**